src/ApiGenerator/Domain/Specification/UrlPath.cs

```diff
@@ -29,7 +29,7 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace ApiGenerator.Domain.Specification 
+namespace ApiGenerator.Domain.Specification
```

Comment:
What happens here?

Reply:
🤷‍♂️ My guess is IntelliJ removing unnecessary space.